libparsec/crates/platform_mountpoint/src/unix/entry_info.rs

```diff
@@ -0,0 +1,50 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use fuser::{FileAttr, FileType};
+
+use crate::{EntryInfo, EntryInfoType};
+
+use super::Inode;
+
+const BLOCK_SIZE: u64 = 512;
+
+impl From<EntryInfoType> for FileType {
+    fn from(value: EntryInfoType) -> Self {
+        match value {
+            EntryInfoType::Dir => Self::Directory,
+            EntryInfoType::File => Self::RegularFile,
+            EntryInfoType::ParsecEntryInfo => Self::RegularFile,
+        }
+    }
+}
+
+impl EntryInfo {
+    pub(super) fn to_file_attr(&self, ino: Inode) -> FileAttr {
```

Comment:
replace `EntryInfo::to_file_attr` by a function that takes `EntryInfo`

Remove this file and put it content in `unix/mod.rs` so that it use and definition is at the same place

Reply:
Done